depends/packages/native_cmake.mk

```diff
@@ -0,0 +1,17 @@
+package=native_cmake
```

Comment:
There needs to be a copyright entry for this new dependency in `contrib/debian/copyright`. (It's [BSD 3-clause with a specific list of contributors](https://github.com/Kitware/CMake/blob/master/Copyright.txt). There's a dependency on LGPL if distributing a binary compiled with Qt, but I don't think that applies to us.)